onnxruntime/contrib_ops/cuda/bert/tensorrt_fused_multihead_attention/flash_attention/sharedCubinLoader.h

```diff
@@ -166,7 +166,7 @@ class TSharedCubinKernelFactory {
     auto const id = hashID(type, sm);
     auto const findIter = mKernels.find(id);
     if (findIter == mKernels.end()) {
-      GSL_SUPPRESS(r .11)
+      GSL_SUPPRESS(r.11)
```

Comment:
Not related to your PR. But this is a case that the code can be improved by leveraging make_unique. So if newKernel->loadCubinKernels() throws, the memory won't leak.

Reply:
thanks for pointing it out